src/app/components/Header/index.jsx

```diff
@@ -34,23 +35,34 @@ const VisuallyHiddenText = styled.span`
   top: 0;
 `;
 
+const StyledSpan = styled.span`
+  padding-bottom: 21px;
```

Comment:
Instead of having a padding-bottom of 21px on this span, could you not have a margin-bottom on the svg instead?

Reply:
@sareh If I add a padding to the svg the anchor tag also inherits this sizing and the clickable area of the anchor tag becomes too long.